src/test/java/uk/ac/ebi/ena/webin/cli/context/genome/GenomeXmlTest.java

```diff
@@ -124,11 +122,9 @@ public void testFastaFile() throws Throwable {
   }
 
   @Test
-  public void testFastaFileAndAgpFile() throws Throwable {
+  public void testFastaFile2() throws Throwable {
```

Comment:
you can remove this test completely if there is another one that tests fasta file.

Reply:
Done.